src/timeline/InputBar.cpp

```diff
@@ -716,6 +755,24 @@ InputBar::video(const QString &filename,
     room->sendMessageEvent(video, mtx::events::EventType::RoomMessage);
 }
 
+QPair<QString, QString>
+InputBar::getCommandAndArgs() const
+{
+    if (!text().startsWith('/'))
```

Comment:
I know the old code did it, but can you put the result of text() into a temp variable here? It makes me a bit anxious everytime a return value of several function calls is used with the same indices :D

Reply:
Sure, I wondered about that myself but figured there was a reason.

Reply:
If there was, I don't remember anymore :D